src/validation.c

```diff
@@ -71,7 +71,7 @@ is_hostname(const char *hostname)
 gboolean
 is_wireguard_key(const char* key)
 {
-    /* Check if this is (most likely) a 265bit, base64 encoded wireguard key */
+    /* Check if this is (most likely) a 256 bit, base64 encoded wireguard key */
```

Comment:
Consider using a more comprehensive approach or a dedicated cryptographic library to validate the WireGuard key. This could enhance security by ensuring the key meets all necessary criteria, not just the basic format checks. [important]